app/src/main/java/com/framgia/moviedb_31/utils/BindingUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.framgia.moviedb_31.utils;
+
+import android.databinding.BindingAdapter;
+import android.support.v7.widget.RecyclerView;
+import android.widget.ImageView;
+import com.bumptech.glide.Glide;
+import com.framgia.moviedb_31.view.homescreen.HomeScreenAdapter;
+
+public class BindingUtils {
+    @BindingAdapter({ "android:moviePoster" })
```

Comment:
```suggestion
    @BindingAdapter({ "imageUrl" })
```
trong một số trường hợp em có thể ignore name space đi cho gọn 
ntn chẳng hạn